docs/source/scancode-toolkit/cli-reference/core-options.rst

```diff
@@ -0,0 +1,282 @@
+`Core` Options
+==============
+
+.. include::  /scancode-toolkit/rst_snippets/basic_options.rst
+
+.. _cli_core:
+
+.. include::  /scancode-toolkit/rst_snippets/core_options.rst
+
+``--generated`` Options
+-----------------------
+
+The ``--generated`` option classifies automatically generated code files with a flag.
+
+An example of using ``--generated`` in a scan::
+
+    ./scancode -clpieu --json-pp sample_generated.json samples --generated
+
+In the results, for each file the following attribute is added with it's corresponding True/False
+value ::
```

Comment:
May be True/False should be lowercase since the JSON output is true/false?

Reply:
Yes. Corrected this. Should I push a single commit with all requested changes?

Reply:
@AyanSinhaMahapatra Yes

Reply:
This is updated.